feat(wysiwyg-editor, rich-text-editor): add wysiwyg component for vue3

emariotti3 opened the pull request:
# feat(wysiwyg-editor, rich-text-editor): add wysiwyg component for vue3

Please refer to [PR ](https://github.com/dialpad/dialtone/pull/113) which adds the wysiwyg component for vue2.

This PR adds the same feature for vue3.

dbecherdialpad reviewed (approved):
is it possible to not generate the documentation changes until after the PR is ready to merge?

hynes-dialpad commented:
> is it possible to not generate the documentation changes until after the PR is ready to merge?

@dbecherdialpad Yes, but we'll have to create a new branch. I've been considering this anyway:

 - `develop` is where everything gets merged from various PRs
 - `production` is the new `master` branch

tedgoas-dialpad commented:
> Update classes to more terse classes

@hynes-dialpad Is this is what we want to do? I'm thinking of the semantic CSS crowd here.

hynes-dialpad commented:
@TedGoas We've already started with terse classes throughout all the type and display utilities. I'd prefer to keep everything the same.

braddialpad committed:
Update CONTRIBUTING.md

braddialpad committed:
docs: move contributing guide to "guides" (#96)

yorbi-dp committed:
fix: video-off icon issue (#99)

braddialpad committed:
docs: fix links starting with / (#91)

braddialpad committed:
fix: inset shadow generation (#94)

juliodialpad committed:
chore: sync figma tokens (#93)

* add sync script

* fix actions

semantic-release-bot committed:
chore(release): dialtone-icons/v4.5.1

## [4.5.1](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.5.0...dialtone-icons/v4.5.1) (2024-01-13)

### Bug Fixes

* video-off icon issue ([#99](https://github.com/dialpad/dialtone/issues/99)) ([4ecb3c0](https://github.com/dialpad/dialtone/commit/4ecb3c0576b69f0fb31911700c1a8dfe6d3a6497))

semantic-release-bot committed:
chore(release): dialtone-css/v8.22.2

## [8.22.2](https://github.com/dialpad/dialtone/compare/dialtone-css/v8.22.1...dialtone-css/v8.22.2) (2024-01-13)

### Bug Fixes

* inset shadow generation ([#94](https://github.com/dialpad/dialtone/issues/94)) ([32cd6b4](https://github.com/dialpad/dialtone/commit/32cd6b4bf64b44da43121c5173f41672c5254813))

### Documentation

* fix old dialpad.design links ([#88](https://github.com/dialpad/dialtone/issues/88)) ([2ec0a6a](https://github.com/dialpad/dialtone/commit/2ec0a6a5150cf51505ba902234061f1c31d2604c))
* update mono-package documentation ([#73](https://github.com/dialpad/dialtone/issues/73)) ([e037c86](https://github.com/dialpad/dialtone/commit/e037c8622c414922ae6d5715b95cd4b9136dcaa1))

semantic-release-bot committed:
chore(release): dialtone-vue3/v3.102.1

## [3.102.1](https://github.com/dialpad/dialtone/compare/dialtone-vue3/v3.102.0...dialtone-vue3/v3.102.1) (2024-01-13)

### Bug Fixes

* icon name validation ([b9cfcff](https://github.com/dialpad/dialtone/commit/b9cfcff6eb23e9a54bb8ee0cde36cc348dd2b606))

semantic-release-bot committed:
chore(release): dialtone-vue2/v2.109.1

## [2.109.1](https://github.com/dialpad/dialtone/compare/dialtone-vue2/v2.109.0...dialtone-vue2/v2.109.1) (2024-01-13)

### Bug Fixes

* icon name validation ([b9cfcff](https://github.com/dialpad/dialtone/commit/b9cfcff6eb23e9a54bb8ee0cde36cc348dd2b606))

semantic-release-bot committed:
chore(release): dialtone/v9.4.1

## [9.4.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.4.0...dialtone/v9.4.1) (2024-01-13)

### Bug Fixes

* icon name validation ([b9cfcff](https://github.com/dialpad/dialtone/commit/b9cfcff6eb23e9a54bb8ee0cde36cc348dd2b606))
* inset shadow generation ([#94](https://github.com/dialpad/dialtone/issues/94)) ([32cd6b4](https://github.com/dialpad/dialtone/commit/32cd6b4bf64b44da43121c5173f41672c5254813))
* token table ([#92](https://github.com/dialpad/dialtone/issues/92)) ([2b0b58e](https://github.com/dialpad/dialtone/commit/2b0b58e4969421c0b6f58ecbfeba14d5b5d00f92))
* video-off icon issue ([#99](https://github.com/dialpad/dialtone/issues/99)) ([4ecb3c0](https://github.com/dialpad/dialtone/commit/4ecb3c0576b69f0fb31911700c1a8dfe6d3a6497))

### Documentation

* fix links starting with / ([#91](https://github.com/dialpad/dialtone/issues/91)) ([b694973](https://github.com/dialpad/dialtone/commit/b6949730abe4732b7fa614991b98f7e4911a5504))
* move contributing guide to "guides" ([#96](https://github.com/dialpad/dialtone/issues/96)) ([10f867c](https://github.com/dialpad/dialtone/commit/10f867cb0dae023c88d83bf21f15b3b445d720fd))

ninamarina committed:
feat(hovercard): add hovercard implementation (#72)

* feat(hovercard): add hovercard implementation

---------

Co-authored-by: Yorbi Barriento <yorbi.barriento@dialpad.com>
Co-authored-by: Julio Ortega <julio.ortega@dialpad.com>

iropolo committed:
fix(emoji-picker): box-sizing (#102)

Co-authored-by: iropolo <ignacio.ropolo@dialpad.com>

semantic-release-bot committed:
chore(release): dialtone-css/v8.22.3

## [8.22.3](https://github.com/dialpad/dialtone/compare/dialtone-css/v8.22.2...dialtone-css/v8.22.3) (2024-01-15)

### Bug Fixes

* **Emoji Picker:** box-sizing ([#102](https://github.com/dialpad/dialtone/issues/102)) ([98f7eb6](https://github.com/dialpad/dialtone/commit/98f7eb68e7c76db295376820d05a32d5950eb817))

semantic-release-bot committed:
chore(release): dialtone-vue3/v3.103.0

# [3.103.0](https://github.com/dialpad/dialtone/compare/dialtone-vue3/v3.102.1...dialtone-vue3/v3.103.0) (2024-01-15)

### Features

* **Hovercard:** add hovercard implementation ([#72](https://github.com/dialpad/dialtone/issues/72)) ([32cc1a8](https://github.com/dialpad/dialtone/commit/32cc1a837043f5fdbf562992ad9bc83f1d2b9252))

semantic-release-bot committed:
chore(release): dialtone-vue2/v2.110.0

# [2.110.0](https://github.com/dialpad/dialtone/compare/dialtone-vue2/v2.109.1...dialtone-vue2/v2.110.0) (2024-01-15)

### Features

* **Hovercard:** add hovercard implementation ([#72](https://github.com/dialpad/dialtone/issues/72)) ([32cc1a8](https://github.com/dialpad/dialtone/commit/32cc1a837043f5fdbf562992ad9bc83f1d2b9252))

semantic-release-bot committed:
chore(release): dialtone/v9.5.0

# [9.5.0](https://github.com/dialpad/dialtone/compare/dialtone/v9.4.1...dialtone/v9.5.0) (2024-01-15)

### Bug Fixes

* **Emoji Picker:** box-sizing ([#102](https://github.com/dialpad/dialtone/issues/102)) ([98f7eb6](https://github.com/dialpad/dialtone/commit/98f7eb68e7c76db295376820d05a32d5950eb817))

### Features

* **Hovercard:** add hovercard implementation ([#72](https://github.com/dialpad/dialtone/issues/72)) ([32cc1a8](https://github.com/dialpad/dialtone/commit/32cc1a837043f5fdbf562992ad9bc83f1d2b9252))

ninamarina committed:
fix(hovercard): timer for vue 2 (#103)

semantic-release-bot committed:
chore(release): dialtone-vue2/v2.110.1

## [2.110.1](https://github.com/dialpad/dialtone/compare/dialtone-vue2/v2.110.0...dialtone-vue2/v2.110.1) (2024-01-15)

### Bug Fixes

* **Hovercard:** timer for vue 2 ([#103](https://github.com/dialpad/dialtone/issues/103)) ([d29c2f6](https://github.com/dialpad/dialtone/commit/d29c2f6804a72119a0da5bada45acfe34ea1784e))

francisrupert committed:
add d-link to each release notes markdown links (#101)

iropolo committed:
feat(datepicker): vue2 style and keyboard navigation enhancement (#95)

* feat: datepicker refactor

---------

Co-authored-by: iropolo <ignacio.ropolo@dialpad.com>

---------

Co-authored-by: iropolo <ignacio.ropolo@dialpad.com>

juliodialpad committed:
fix: nx affected dependencies (#104)

semantic-release-bot committed:
chore(release): dialtone-tokens/v1.23.5

## [1.23.5](https://github.com/dialpad/dialtone/compare/dialtone-tokens/v1.23.4...dialtone-tokens/v1.23.5) (2024-01-16)

### Bug Fixes

* nx affected dependencies ([#104](https://github.com/dialpad/dialtone/issues/104)) ([09ea0bb](https://github.com/dialpad/dialtone/commit/09ea0bb8c9756dea12e0a03b454df7b942a6e340))

### Documentation

* fix old dialpad.design links ([#88](https://github.com/dialpad/dialtone/issues/88)) ([2ec0a6a](https://github.com/dialpad/dialtone/commit/2ec0a6a5150cf51505ba902234061f1c31d2604c))

semantic-release-bot committed:
chore(release): dialtone-tokens/v1.23.5 gradle

semantic-release-bot committed:
chore(release): dialtone-icons/v4.5.2

## [4.5.2](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.5.1...dialtone-icons/v4.5.2) (2024-01-16)

### Bug Fixes

* nx affected dependencies ([#104](https://github.com/dialpad/dialtone/issues/104)) ([09ea0bb](https://github.com/dialpad/dialtone/commit/09ea0bb8c9756dea12e0a03b454df7b942a6e340))

semantic-release-bot committed:
chore(release): dialtone-css/v8.23.0

# [8.23.0](https://github.com/dialpad/dialtone/compare/dialtone-css/v8.22.3...dialtone-css/v8.23.0) (2024-01-16)

### Bug Fixes

* nx affected dependencies ([#104](https://github.com/dialpad/dialtone/issues/104)) ([09ea0bb](https://github.com/dialpad/dialtone/commit/09ea0bb8c9756dea12e0a03b454df7b942a6e340))

### Features

* **Datepicker:** vue2 style and keyboard navigation enhancement ([#95](https://github.com/dialpad/dialtone/issues/95)) ([2850eae](https://github.com/dialpad/dialtone/commit/2850eae8b1a54b6886fcc8c7cd0bee1f4f2c79be))

semantic-release-bot committed:
chore(release): dialtone-vue3/v3.104.0

# [3.104.0](https://github.com/dialpad/dialtone/compare/dialtone-vue3/v3.103.0...dialtone-vue3/v3.104.0) (2024-01-16)

### Bug Fixes

* nx affected dependencies ([#104](https://github.com/dialpad/dialtone/issues/104)) ([09ea0bb](https://github.com/dialpad/dialtone/commit/09ea0bb8c9756dea12e0a03b454df7b942a6e340))

### Features

* **Datepicker:** vue2 style and keyboard navigation enhancement ([#95](https://github.com/dialpad/dialtone/issues/95)) ([2850eae](https://github.com/dialpad/dialtone/commit/2850eae8b1a54b6886fcc8c7cd0bee1f4f2c79be))

semantic-release-bot committed:
chore(release): eslint-plugin-dialtone/v1.0.1

## [1.0.1](https://github.com/dialpad/dialtone/compare/eslint-plugin-dialtone/v1.0.0...eslint-plugin-dialtone/v1.0.1) (2024-01-16)

### Bug Fixes

* nx affected dependencies ([#104](https://github.com/dialpad/dialtone/issues/104)) ([09ea0bb](https://github.com/dialpad/dialtone/commit/09ea0bb8c9756dea12e0a03b454df7b942a6e340))

### Documentation

* fix old dialpad.design links ([#88](https://github.com/dialpad/dialtone/issues/88)) ([2ec0a6a](https://github.com/dialpad/dialtone/commit/2ec0a6a5150cf51505ba902234061f1c31d2604c))

semantic-release-bot committed:
chore(release): dialtone-vue2/v2.111.0

# [2.111.0](https://github.com/dialpad/dialtone/compare/dialtone-vue2/v2.110.1...dialtone-vue2/v2.111.0) (2024-01-16)

### Bug Fixes

* nx affected dependencies ([#104](https://github.com/dialpad/dialtone/issues/104)) ([09ea0bb](https://github.com/dialpad/dialtone/commit/09ea0bb8c9756dea12e0a03b454df7b942a6e340))

### Features

* **Datepicker:** vue2 style and keyboard navigation enhancement ([#95](https://github.com/dialpad/dialtone/issues/95)) ([2850eae](https://github.com/dialpad/dialtone/commit/2850eae8b1a54b6886fcc8c7cd0bee1f4f2c79be))

semantic-release-bot committed:
chore(release): dialtone/v9.6.0

# [9.6.0](https://github.com/dialpad/dialtone/compare/dialtone/v9.5.0...dialtone/v9.6.0) (2024-01-16)

### Bug Fixes

* **Hovercard:** timer for vue 2 ([#103](https://github.com/dialpad/dialtone/issues/103)) ([d29c2f6](https://github.com/dialpad/dialtone/commit/d29c2f6804a72119a0da5bada45acfe34ea1784e))
* nx affected dependencies ([#104](https://github.com/dialpad/dialtone/issues/104)) ([09ea0bb](https://github.com/dialpad/dialtone/commit/09ea0bb8c9756dea12e0a03b454df7b942a6e340))

### Features

* **Datepicker:** vue2 style and keyboard navigation enhancement ([#95](https://github.com/dialpad/dialtone/issues/95)) ([2850eae](https://github.com/dialpad/dialtone/commit/2850eae8b1a54b6886fcc8c7cd0bee1f4f2c79be))

juliodialpad committed:
chore: make releasing linear to avoid race conditions

yorbi-dp committed:
docs: adjusted background on utilities examples (#97)

yorbi-dp committed:
docs: added icons section in design language (#50)

* docs: added icons section in design language

v1 of the new documentation

* added examples and fixed content

* updated version from feedback

* small changes

- change the example in the design/icons view. - added a link to the Catalog in the icon component

* update from feedback

* update icon component page from feedback

* small update in content

* site nav update

ninamarina committed:
Update commit convention link in CONTRIBUTING.md

ninamarina committed:
Update commit convention links in CONTRIBUTING.md

yorbi-dp committed:
docs: added update url with search value (#100)

* docs: added update url with search value

* docs: added copy the link to a popover

* update button active and url changes

* fix to avoid scroll animation

* fix for code review

* fix for mobile

* fixed space with title

---------

Co-authored-by: Nina <ninamrepetto@gmail.com> 🥳

braddialpad committed:
fix(popover): set text color within popover dialog (#108)

francisrupert committed:
corrected selectmenu's icon color in dark mode (#107)

yorbi-dp committed:
fix: Icons Update (#110)

* fix: update icons from feedback

* delete end line in keywords

braddialpad committed:
fix(feed-item-row): spacing and transition improvements - Vue 2 (#109)

semantic-release-bot committed:
chore(release): dialtone-icons/v4.5.3

## [4.5.3](https://github.com/dialpad/dialtone/compare/dialtone-icons/v4.5.2...dialtone-icons/v4.5.3) (2024-01-18)

### Bug Fixes

* Icons Update ([#110](https://github.com/dialpad/dialtone/issues/110)) ([4804d83](https://github.com/dialpad/dialtone/commit/4804d83f29cd52960fc2c9ab67f2ae37635951e0))

semantic-release-bot committed:
chore(release): dialtone-css/v8.23.1

## [8.23.1](https://github.com/dialpad/dialtone/compare/dialtone-css/v8.23.0...dialtone-css/v8.23.1) (2024-01-18)

### Bug Fixes

* **Popover:** set text color within popover dialog ([#108](https://github.com/dialpad/dialtone/issues/108)) ([9ea0fe7](https://github.com/dialpad/dialtone/commit/9ea0fe715cff346ba1bf4b35ba1772a6c998a397))

semantic-release-bot committed:
chore(release): dialtone-vue2/v2.111.1

## [2.111.1](https://github.com/dialpad/dialtone/compare/dialtone-vue2/v2.111.0...dialtone-vue2/v2.111.1) (2024-01-18)

### Bug Fixes

* **Feed Item Row:** spacing and transition improvements - Vue 2 ([#109](https://github.com/dialpad/dialtone/issues/109)) ([b33a82c](https://github.com/dialpad/dialtone/commit/b33a82c68f27d0db8760e4332667fa19eacd5b24))

semantic-release-bot committed:
chore(release): dialtone/v9.6.1

## [9.6.1](https://github.com/dialpad/dialtone/compare/dialtone/v9.6.0...dialtone/v9.6.1) (2024-01-18)

### Bug Fixes

* **Feed Item Row:** spacing and transition improvements - Vue 2 ([#109](https://github.com/dialpad/dialtone/issues/109)) ([b33a82c](https://github.com/dialpad/dialtone/commit/b33a82c68f27d0db8760e4332667fa19eacd5b24))
* Icons Update ([#110](https://github.com/dialpad/dialtone/issues/110)) ([4804d83](https://github.com/dialpad/dialtone/commit/4804d83f29cd52960fc2c9ab67f2ae37635951e0))
* **Popover:** set text color within popover dialog ([#108](https://github.com/dialpad/dialtone/issues/108)) ([9ea0fe7](https://github.com/dialpad/dialtone/commit/9ea0fe715cff346ba1bf4b35ba1772a6c998a397))

### Documentation

* added icons section in design language ([#50](https://github.com/dialpad/dialtone/issues/50)) ([fe7a0ac](https://github.com/dialpad/dialtone/commit/fe7a0ac949f57de8b755c52e99a1b92096b727fd))
* added update url with search value ([#100](https://github.com/dialpad/dialtone/issues/100)) ([174b3fc](https://github.com/dialpad/dialtone/commit/174b3fc52c2351855e5e6ac5a5bddae58d55094e))
* adjusted background on utilities examples ([#97](https://github.com/dialpad/dialtone/issues/97)) ([43cb609](https://github.com/dialpad/dialtone/commit/43cb609803588055cfdfd81fbfd9a4a5ddaf4e35))

emariotti3 committed:
feat(wysiwyg-editor, rich-text-editor): add wysiwyg component

emariotti3 committed:
feat(wysiwyg-editor, rich-text-editor): add wysiwyg component

iropolo committed:
fix: input focus within (#106)

Co-authored-by: iropolo <ignacio.ropolo@dialpad.com>

emariotti3 committed:
feat(wysiwyg-editor, rich-text-editor): add wysiwyg component

emariotti3 committed:
feat(wysiwyg-editor, rich-text-editor): add dependencies to vue3 package.json

emariotti3 committed:
feat(wysiwyg-editor, rich-text-editor): add wysiwyg component for vue3

emariotti3 committed:
feat(wysiwyg-editor, rich-text-editor): remove changes related to vue2
